Sections/US-States/MT/GPP Extension: IAB Privacy’s Montana Privacy Technical Specification.md

```diff
@@ -0,0 +1,232 @@
+<h1 id="gpp-extension-iab-privacy-s-montana-privacy-technical-specification">GPP Extension: IAB Privacy’s Montana Privacy Technical Specification</h1>
+<h2 id="about-this-document">About this document</h2>
+<p>The global standard <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform">GPP</a> defines a way for local standards to &quot;plug-in&quot; into the existing mechanics defined by GPP and the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Core/CMP%20API%20Specification.md">GPP client side API</a>. This document outlines the technical specification for using the GPP specifications with the IAB Privacy Multi-State Privacy Agreement legal requirements.</p>
```

Comment:
This appears wrong. Connecticut for example says:  "This document outlines the technical specification for using the Connecticut section of the GPP specification by those who (i) are Signatories to IAB Privacy, Inc.’s [Multi-State Privacy Agreement (MSPA)](https://www.iabprivacy.com/); and (ii) those who are not signatories of the MSPA."

Reply:
@lamrowena cc

Reply:
Would this work as revised language?

"This document outlines the technical specification for using the [_INSERT STATE_] section of the GPP specifications in accordance with the IAB Privacy Multi-State Privacy Agreement legal requirements, applicable to both Signatories and non-Signatories of the MSPA."

Reply:
yes